Formula/hetzner-kube.rb

```diff
@@ -0,0 +1,23 @@
+class HetznerKube < Formula
+  desc "CLI tool for provisioning kubernetes clusters on Hetzner Cloud"
+  homepage "https://github.com/xetys/hetzner-kube"
+  url "https://github.com/xetys/hetzner-kube/archive/0.3.tar.gz"
+  sha256 "300ad82fdeeb5f68d034571e1269025a2da73cd55be34a1cdb7f09257b1807af"
+
+  depends_on "go" => :build
+
+  def install
+    ENV["GOPATH"] = buildpath
+
+    bin_path = buildpath/"src/github.com/xetys/hetzner-kube"
+    bin_path.install Dir["*"]
+
+    cd bin_path do
+      system "go", "build", "-o", bin/"hetzner-kube", "."
+    end
+  end
+
+  test do
+    assert_match version.to_s, shell_output("#{bin}/hetzner-kube version")
```

Comment:
We need a test that exercises the some of the functionality of the app.

Reply:
@fxcoudert as this is for managing remote system clusters it's difficult to include real tests of functionality the here. Would testing against empty output of the `hetzner-kube cluster list` command like be sufficient?